cfme/test_framework/sprout/plugin.py

```diff
@@ -73,8 +80,29 @@ def mangle_in_sprout_appliances(config):
     appliances = config.option.appliances
     log.info("Appliances were provided:")
     for appliance in requested_appliances:
+
         url = "https://{}/".format(appliance["ip_address"])
```

Comment:
**REQUIRED**: Can you change this to pull the URL off appliance using the `.url` property?

Reply:
honestly, it isn't related to this pr but I moved it to Appliance class